### ISSUE TITLE
Unable to authorize

### ISSUE THREAD

**Issue by oblivia-simplex** [gavvvr/obsidian-imgur-plugin#14]
Hi there,
I'm unable to authorize the plugin to upload to my imgur account. This is the error message that Imgur gives me:

```
data |  
-- | --
error | "Unauthorized"
request | "/oauth2/authorize"
method | "POST"
success | false
status | 403
```

I've tried this both before and after adding my Imgur ClientId to the ClientId field in the plugin's data.json.

**Comment by gavvvr** [comment]
Hi @oblivia-simplex

> before and after adding my Imgur ClientId to the ClientId field in the plugin's data.json.

First thing I want to emphasise: you are not expected to edit `data.json` manually. As a user, you are not even expected to know it exists.

---

I just checked authentication works for me. Steps I did
- installed the plugin
- enabled the plugin
- opened it's settings
- chosen 'Authenticated Imgur upload' from dropdown list
- The modal window appeared with instructions appeared and browser had been opened in several seconds, where I entered my credentials
- After that the browser calls `obsidian://...` to pass access token to it. That's it. Authentication successful

What did you do differently? Where did you copy the error from? One more thing to check: make sure your Imgur credentials (password) are valid
